### ISSUE TITLE
Deprecation warning by `wp-env`

### ISSUE THREAD

**Issue by ernilambar** [10up/plugin-check#1236]
After upgrading `@wordpress/env` (v11+), running `wp-env start` prints output like:

```
Warning: wp-env starts both development and tests environments by default.
This behavior is deprecated and will be removed in a future version.
To avoid this warning, add "testsEnvironment": false to your .wp-env.json.
The "env", "testsPort", and "testsEnvironment" options are also deprecated.
Use the --config option with a separate config file for test environments instead.
```
